### PR TITLE
Throw exception if collection is not found in db

### PR DESCRIPTION
**Pull request by Akrion**
Throw if the collection is not there

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,15 +1,18 @@
-var Promise = require('bluebird')
+var Promise = require('bluebird'),
+    _ = require('lodash')
```
Comment (daedalus28) [inline src/index.js]: Use fp :)

File: src/index.js [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 var mongoDSL = (client, dsl) => {
-  var Collection = client.collection(dsl.collection)
-
-  // if (dsl.resultOptions)
-  //     return Collection.find(dsl.criteria, dsl.resultOptions)
-  // if (dsl.count)
-  //     return Collection.count(dsl.criteria)
-
-  if (dsl.aggs) return Collection.aggregate(dsl.aggs).toArray()
+  if (_.get(client, `collections.${dsl.collection}`)) {
```
Comment (daedalus28) [inline src/index.js]: Is this the same as client.collection? If so, you could just assign it and throw right away if it’s missing

**Comment by daedalus28** [comment]
Don’t forget the change log and package changes. Duti should have commented actually, not sure why it didn’t

**Comment by Akrion** [comment]
Done with the requested changes

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Your PR has lint errors. Please fix these and commit them.
      
<details>
  <summary>date.js</summary>
  <p>Full Path: <code>/home/circleci/project/src/example-types/date.js</code></p>
  
  <strong>Expected method shorthand.</strong>
  <code>Line 7: filter: context => {</code>
</details>
<details>
  <summary>results.js</summary>
  <p>Full Path: <code>/home/circleci/project/src/example-types/results.js</code></p>
  
  <strong>Expected method shorthand.</strong>
  <code>Line 5: result: (context, search) => {</code>

  <strong>Expected property shorthand.</strong>
  <code>Line 43: results: results,</code>
</details>
<details>
  <summary>index.js</summary>
  <p>Full Path: <code>/home/circleci/project/src/index.js</code></p>
  
  <strong>Split 'var' declarations into multiple statements.</strong>
  <code>Line 1: var Promise = require('bluebird'),</code>

  <strong>Expected method shorthand.</strong>
  <code>Line 21: runSearch: (options, context, schema, filters, aggs) => {</code>
</details>
<details>
  <summary>text.js</summary>
  <p>Full Path: <code>/home/circleci/project/test/example-types/text.js</code></p>
  
  <strong>Expected property shorthand.</strong>
  <code>Line 33: operator: operator,</code>
</details>
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  You should add at least 1 more reviewer to the PR
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  The README has not been updated. Please update the README.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  We were able to automatically fix some formatting issues in this PR for you!
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**